src/PixiEditor/Data/Localization/Languages/en.json

```diff
@@ -426,6 +426,8 @@
     "BROWSE_PALETTES": "Browse palettes",
     "LOAD_PALETTE": "Load palette",
     "SAVE_PALETTE": "Save palette",
+    "DISCARD_PALETTE": "Discard palette",
+    "DISCARD_PALETTE_CONFIRMATION": "Are you sure you want to discard current palette? This cannot be undone.",
```

Comment:
I think this sounds better

Are you sure you want to discard *__the__* current palette? This cannot be undone.

Reply:
For me it sounds wierd, however this can be changed in POEditor